docs/testing/e2e-plan.md

```diff
@@ -0,0 +1,48 @@
+# Curated Packages End-To-End Testing Plan
+
+This document summarizes the current state and future plans for end-to-end (E2E) testing in the EKS-A Curated Packages (CP) project.
+
+## The Present
+
+End-to-end tests can be written to exercise CP workflows. Several parameters can be selected to control the environment (cluster) in which a test runs. These include: EKS-A provider, node OS, and Kubernetes (K8s) version. For examples, see [TestCPackagesHarborInstallSimpleFlow](https://github.com/aws/eks-anywhere/blob/de84797d702028e15deb3363bbe222489f00229d/test/e2e/harbor_test.go) and [TestCPackagesVSphereKubernetes121SimpleFlow](https://github.com/aws/eks-anywhere/blob/de84797d702028e15deb3363bbe222489f00229d/test/e2e/curated_packages_test.go#L102).
```

Comment:
Let's change the "These include" for an actual list and let's try to make it as exhaustive as possible. I'd add `eks-anywhere version`.

Reply:
Ah, well, see, this was meant to summarize the current state, and in the current state, the "framework" as it were doesn't support different eks-a versions. I'm not sure, but i think it would take a lot hoop-jumping and manual installation. This is a side-effect of the packages e2e tests running from the eks-a repo.